src/website/apps/blog/tests.py

```
@@ -231,4 +244,675 @@ def test_calendar_endpoint_handles_legacy_quill_data(self):
+        )
+        self.assertEqual(link.event, event)
+        self.assertEqual(link.organizer, org)
+        self.assertEqual(link.role, 'organizer')
+        self.assertTrue(event.is_side_event is False)
+        self.assertEqual(event.event_organizer_links.count(), 1)
+
+    def test_duplicate_organizer_link_is_rejected_at_db_level(self):
+        event = self._make_event('AirQo Clean Air Week 2026')
+        org = self._make_organizer('AirQo')
+        EventOrganizer.objects.create(event=event, organizer=org)
+        with self.assertRaises(Exception):
```

Ruff (0.15.15)

[warning] Do not assert blind exception: `Exception`

(B017)